diff --git a/tracks/4- cloud.yml b/tracks/4- cloud.yml
index 67a9d36..6a59def 100644
--- a/tracks/4- cloud.yml       
+++ b/tracks/4- cloud.yml       
@@ -60,3 +60,11 @@ sessions:
       about: "Mohamed is the GraalVM Sustaining Manager at Oracle Labs, Oracle’s Research Center in Casablanca.
 Mohamed is managing the global sustaining efforts for GraalVM Project, through his career he worked in different IT roles and co-founded multiple startups, He is also deeply involved with local tech communities." 
       avatar:  https://lh3.googleusercontent.com/pw/ACtC-3cMd71zGaazM_u43URfX7gNMvEr7JVIAOCYmI99jiLnBATkNQxc5-4oBvz4XE89uYOk9GwZgpUA7YNspOrIBGxg_ScRtY79QCDhnTEjMfSvEXgSHj5wYWuw-aJ7DraXI4WXYhcRrUJv8OaKv2wbZxQvTg=w842-h1122-no?authuser=0
+  - title: Kubernetes VS Docker Swarm
+    description: "I will draw comparisons between Kubernetes and Docker Swarm. I will possibly include simple demos. Participants will benefit from knowing what to pick between these two technologies and when." 
+    time: "time: N/A"
+    speaker:
+      name: Zakaria Boualaid
+      role: Senior DevOps Engineer 
+      about: "Senior DevOps Engineer at Nimbleways, former DevOps Engineer at Oracle. I am passionate about technology and work ethic." 
+      avatar: /speakers/boualaid.jpg